### ISSUE TITLE
Hangs when multiple targets specified

### ISSUE THREAD

**Issue by jsjohns** [nabla-c0d3/sslyze#259]
I'm running into trouble running SSLyze against multiple targets. Specifically, SSLyze hangs without outputting any sort of result (XML, JSON, and command-line output formats tested) when specifying 10 random addresses via the `--target_in` argument. 

I see this behavior with SSLyze v1.1.4 and Python 3.6.3 on Ubuntu 16.04 and macOS 10.13.1.

```
$ cat targets.txt
104.72.217.21:443
64.6.109.50:443
207.99.100.218:443
72.240.119.246:443
54.225.80.71:443
207.219.182.120:443
104.16.89.172:443
35.227.246.79:443
99.138.64.210:443
128.136.49.36:443

$ sslyze --quiet --regular --json_out=output.json --targets_in=targets.txt
<... hangs ...>
```

Omitting the `--targets_in` arguments gives an explicit error:

```
$ sslyze --regular 104.72.217.21:443 64.6.109.50:443 207.99.100.218:443 72.240.119.246:443 54.225.80.71:443 207.219.182.120:443 104.16.89.172:443 35.227.246.79:443 99.138.64.210:443 128.136.49.36:443



 AVAILABLE PLUGINS
 -----------------

  HttpHeadersPlugin
  SessionRenegotiationPlugin
  FallbackScsvPlugin
  CertificateInfoPlugin
  OpenSslCipherSuitesPlugin
  SessionResumptionPlugin
  CompressionPlugin
  HeartbleedPlugin
  OpenSslCcsInjectionPlugin



 CHECKING HOST(S) AVAILABILITY
 -----------------------------

   104.16.89.172:443                       => 104.16.89.172
   207.219.182.120:443                       => 207.219.182.120
   54.225.80.71:443                       => 54.225.80.71
   128.136.49.36:443                       => 128.136.49.36
   72.240.119.246:443                       => 72.240.119.246
Traceback (most recent call last):
  File "/usr/local/bin/sslyze", line 9, in <module>
    load_entry_point('SSLyze==1.1.4', 'console_scripts', 'sslyze')()
  File "/usr/local/lib/python3.5/dist-packages/sslyze/__main__.py", line 93, in main
    global_scanner.queue_scan_command(server_connectivity_info, scan_command)
  File "/usr/local/lib/python3.5/dist-packages/sslyze/concurrent_scanner.py", line 94, in queue_scan_command
    self._check_and_create_process(server_info.hostname)
  File "/usr/local/lib/python3.5/dist-packages/sslyze/concurrent_scanner.py", line 123, in _check_and_create_process
    self._hostname_queues_dict[hostname] = random.choice(self._hostname_queues_dict.values())
  File "/usr/lib/python3.5/random.py", line 256, in choice
    return seq[i]
TypeError: 'dict_values' object does not support indexing
```

**Comment by gopar** [comment]
The 2nd error should be fixed in `1.1.5`

**Comment by nabla-c0d3** [comment]
Thanks @gopar - @jsjohns can you confirm?
